documentation/tutorial/02-environment-setup.md

```diff
@@ -0,0 +1,74 @@
+# Environment Setup
+
+This is part of the
+[Charmed Superset Tutorial]().
+Please refer to this page for more information and the overview of the content.
+
+## Setting up MicroK8s
+
+Charmed Superset relies on Microk8s or Kubernetes as a container orchestration system.
+We'll now set up [MicroK8s](https://microk8s.io/docs) as follows: 
+
+On your machine, install and configure MicroK8s:
+
+```bash
```

Comment:
As aforementioned, let's remove code comments and add text instead for a better reading